docs/getting_started/hopsworksai/guides/getting_started_azure.rst

```diff
@@ -173,7 +173,86 @@ Congratulations, you have successfully connected you Azure account to Hopsworks.
     :align: center
     :figclass: align-center
 
-Step 2: Deploying a Hopsworks cluster
+Step 2: Creating and configuring a storage
+------------------------------------------
+The hopsworks clusters deployed by hopsworks.ai store their data in a container in your Azure accoutn.
+To enable this you need to create a storage account and a User Assigned Managed Identity to give the hopsworks cluster access to the storage.
+
+Step 2.1: Creating a User Assigned Managed Identity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Proceed to the Azure Portal and open the Resource Group that you want to use for Hopsworks.ai. Click on *Add*.
+
+.. _add-to-resource-group.png: ../../../_images/add-to-resource-group.png
+.. figure:: ../../../imgs/hopsworksai/add-to-resource-group.png
+    :alt: Add to resource group.
+    :target: `add-to-resource-group.png`_
+    :align: center
+    :figclass: align-center
+    :borderWidth: 0.5
+    :borderColor:"black"
+
+
+Search for *User Assigned Managed Identity* and click on it.
+
+.. _search-user-assigned-identity.png: ../../../_images/search-user-assigned-identity.png
+.. figure:: ../../../imgs/hopsworksai/search-user-assigned-identity.png
+    :alt: Search User Assigned Managed Identity.
+    :target: `search-user-assigned-identity.png`_
+    :align: center
+    :figclass: align-center
+
+Click on *Create*. Then, select the Location you want to use and give a name to the identity. Click on *Review + create*. Finally click on *Create*.
+
+.. _create-user-assigned-identity.png: ../../../_images/create-user-assigned-identity.png
+.. figure:: ../../../imgs/hopsworksai/create-user-assigned-identity.png
+    :alt: Create a User Assigned Managed Identity.
+    :target: `create-user-assigned-identity.png`_
+    :align: center
+    :figclass: align-center
+
+Step 2.2: Creating a Storage account
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Proceed to the Azure Portal and open the Resource Group that you want to use for Hopsworks.ai. Click on *Add*.
+
+.. _add-to-resource-group2.png: ../../../_images/add-to-resource-group.png
+.. figure:: ../../../imgs/hopsworksai/add-to-resource-group.png
+    :alt: Add to resource group.
+    :target: `add-to-resource-group.png`_
+    :align: center
+    :figclass: align-center
+
+Search for *Storage account* and click on it.
+
+.. _search-storage-account.png: ../../../_images/search-storage-account.png
+.. figure:: ../../../imgs/hopsworksai/search-user-assigned-identity.png
+    :alt: Search Storage Account Identity.
+    :target: `search-storage-account.png`_
+    :align: center
+    :figclass: align-center
+
+Click on *Create*. Then, give a name to your storage account, select the Location you want to use and click on *Review + create*. Finally click on *Create*.
```

Comment:
Click on *Create*, name your storage account, select the Location you want to use and click on *Review + create*.